README.md

```diff
@@ -21,6 +21,7 @@
 
 - [Install](#install)
 - [Usage](#usage)
+  - [Setup](#setup)
```

Comment:
are you planning to add the the other titles that you defined in the midsection here as well?